CustomerRegister/Repositories/CustomerService.cs

```diff
@@ -0,0 +1,73 @@
+﻿
+
+using CustomerRegister.Repositories.Interfaces;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace CustomerRegister
+{
+    public class CustomerService : ICustomerService
+
+    {
+        private readonly List<CustomerEntity> _customers = new();
+
+        public bool Exists(CustomerEntity selectedCustomer)
+        {
+            var response = _customers.Any(customer => customer.Cpf == selectedCustomer.Cpf || customer.Email == selectedCustomer.Email) && (selectedCustomer.Id != selectedCustomer.Id);
+            return response;
+        }
+
+        public bool DuplicatedRegister(CustomerEntity selectedCustomer)
+        {
+            if(_customers.Any(customer => customer.Cpf == selectedCustomer.Cpf || customer.Email == selectedCustomer.Email))
+                {
+                return true;
+                }
+            return false;
```

Comment:
```suggestion
            if(_customers.Any(customer => customer.Cpf == selectedCustomer.Cpf || customer.Email == selectedCustomer.Email))
                return true;

            return false;
```
Pode ser feito como sugerido acima. Entretanto, eu faria um If para lidar com cpf ja existente e outro para email.